helm-charts/helm2/strimzi-kafka-operator/templates/050-ConfigMap-strimzi-cluster-operator.yaml

```diff
@@ -26,3 +26,6 @@ data:
     logger.zookeepertrustmanager.name = org.apache.zookeeper
     logger.zookeepertrustmanager.level = WARN
     logger.zookeepertrustmanager.additivity = false
+
+    # Netty is by default on DEBUG level, which returns not related exceptions and errors -> we set it to INFO
```

Comment:
I do not think this is correct. You switch Netty to debug by setting the root logger to debug, or? This just keeps a separate level and will not be changed by the root logger.